Modelica/Mechanics/MultiBody/Sensors/CutForceAndTorque.mo

```diff
@@ -3,13 +3,17 @@ model CutForceAndTorque "Measure cut force and cut torque vector"
 
   import Modelica.Mechanics.MultiBody.Types;
 
-  Modelica.Blocks.Interfaces.RealOutput force[3](each final quantity="Force", each final unit="N")
```

Comment:
This line is only formatting, right?

Reply:
Yes, to have the same formatting for that in all models.